src/main/java/seedu/duke/command/SeeAllIncomesCommand.java

```diff
@@ -0,0 +1,62 @@
+package seedu.duke.command;
+
+import seedu.duke.financial.FinancialEntry;
+import seedu.duke.financial.FinancialList;
+import seedu.duke.financial.Income;
+
+import java.time.LocalDate;
+
+/**
+ * Command to print all incomes recorded in the financial list.
+ */
+public class SeeAllIncomesCommand extends Command {
+    private LocalDate start;
+    private LocalDate end;
```

Comment:
The variable name might be a bit ambiguous, maybe it could be better refined as `startDate`?